From akpm@linux-foundation.org  Tue May  4 13:04:49 2010
From: Geert Uytterhoeven <geert@linux-m68k.org>
Date: Tue, 27 Apr 2010 14:16:40 -0700
Subject: serial167: kill unused variables
To: greg@kroah.com
Cc: akpm@linux-foundation.org, geert@linux-m68k.org, alan@lxorguk.ukuu.org.uk
Message-ID: <201004272116.o3RLGexn020408@imap1.linux-foundation.org>


From: Geert Uytterhoeven <geert@linux-m68k.org>

commits 638157bc1461f6718eeca06bedd9a09cf1f35c36 ("serial167: prepare to
push BKL down into drivers") and 4165fe4ef7305609a96c7f248cefb9c414d0ede5
("tty: Fix up char drivers request_room usage") removed code without
removing the corresponding variables:

| drivers/char/serial167.c: In function 'cd2401_rx_interrupt':
| drivers/char/serial167.c:630: warning: unused variable 'len'
| drivers/char/serial167.c: In function 'cy_ioctl':
| drivers/char/serial167.c:1531: warning: unused variable 'val'

Remove the variables to kill the warnings.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
Cc: Alan Cox <alan@lxorguk.ukuu.org.uk>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/char/serial167.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/char/serial167.c
+++ b/drivers/char/serial167.c
@@ -627,7 +627,6 @@ static irqreturn_t cd2401_rx_interrupt(i
 	char data;
 	int char_count;
 	int save_cnt;
-	int len;
 
 	/* determine the channel and change to that context */
 	channel = (u_short) (base_addr[CyLICR] >> 2);
@@ -1528,7 +1527,6 @@ static int
 cy_ioctl(struct tty_struct *tty, struct file *file,
 	 unsigned int cmd, unsigned long arg)
 {
-	unsigned long val;
 	struct cyclades_port *info = tty->driver_data;
 	int ret_val = 0;
 	void __user *argp = (void __user *)arg;
